lib/material/internal/material_util.flow

```diff
@@ -1117,7 +1121,27 @@ mColorLerp(previous : MColor, current : MColor, percent : double) -> MColor {
 }
 
 // Navigate through list by arrows, not by tab. Wrap each item of list into this.
```

Comment:
Move the comment back to MArrowNavigationItem and add the proper one to this function